static/css/tailwind.src.css

```
@@ -6,6 +6,55 @@
 @tailwind base;
 @tailwind components;
 
+@layer base {
+
+  h1 {
+    font-size: 70px;
+    line-height: 75.5px;
+    letter-spacing: 0.5px;
+    font-weight: 700; 
```

This is just an example of how you could do it with tailwind:
```suggestion css
h1 {
  @apply 
     font-h1
     leading-h1   
     tracking-wide
     font-bold;
}
```
We would proceed to defined the exact values for `font-h1` and `leading-h1` in `tailwind.config.js`.

---

That's clean